src/index.html

```diff
@@ -1,12 +1,412 @@
+            <a href="#Smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#Laptops&computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >Laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <section class="card card--1" data-qa="card">
+
+        <img
+          src="images/imac.jpeg"
+          alt="imacIcon"
+          class="card__icon"
+        >
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__product">
+          Product code: 195434
+        </p>
+
+
+        <div  class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__review">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <p class="card__word">Price:</p>
+          <p class="card__digit">$2,199</p>
+        </div>
+
+        <div class="card__link">
+          <a
+            class="card__item"
+            href="#buy"
+            data-qa="card-hover"
+          >
+            Buy
+
+          </a>
+        </div>
+        </div>
+      </section>
+
+      <section class="card card--2">
+
+        <img
+          src="images/imac.jpeg"
+          alt="imacIcon"
+          class="card__icon"
+        >
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__product">
+          Product code: 195434
+        </p>
+
+
+        <div  class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__review">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <p class="card__word">Price:</p>
+          <p class="card__digit">$2,199</p>
+        </div>
+
+        <div class="card__link">
+          <a
+            class="card__item"
+            href="#buy"
+            data-qa="card-hover"
+          >
+            Buy
+
+          </a>
+        </div>
+        </div>
+      </section>
```

Comment:
this two divs on the same lines, looks strange

Reply:
all usless <div's were removed